### PR TITLE
Dashboard UI

### PR DESCRIPTION
**Pull request by syriazapanta**

### CODE REVIEW

**Comment by princeryansy** [comment]
Refrain from including html tags in lang files as this will be truly hard to manage in the future.

File: application/models/ApplicationsModel.php [hunk 1]
```diff
@@ -175,4 +189,17 @@ public function getApplicantJob()
             ->get()
             ->row_array();
     }
+
+    public function getNewApplicants()
```
Comment (princeryansy) [inline application/models/ApplicationsModel.php]: the method returns a count, please change method name to a more appropriate one.

File: application/modules/hirer/controllers/Announcement.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ public function index($id = 0)
                     ->getById('title', 'story', 'date_created')
             )
         );
-
+
```
Comment (princeryansy) [inline application/modules/hirer/controllers/Announcement.php]: please remove tab space

File: application/modules/hirer/views/announcement.php [hunk 3]
```diff
@@ -20,18 +20,20 @@
...
+                        <h6>
+                            <?php
+                            $total = count($announcements);
 
-                        for ($i = 0;$i < $total; $i++) {
-                            echo '<a href="' . url_hirer('announcement/' . $announcements[$i]['id']. '') . '"><span class="badge badge-light text-danger m-1"> ' . $announcements[$i]['title'] . '</span></a>';
-                        }
-                        ?>
-                    </h6>
+                            for ($i = 0;$i < $total; $i++) {
```
Comment (princeryansy) [inline application/modules/hirer/views/announcement.php]: why not use foreach.

File: application/modules/hirer/views/home.php [hunk 4]
```diff
@@ -1,2 +1,113 @@
 <?php
-echo $dashboardNote;
+?>
```
Comment (princeryansy) [inline application/modules/hirer/views/home.php]: remove php tag if it is empty.

File: application/modules/hirer/views/home.php [hunk 5]
```diff
@@ -1,2 +1,113 @@
...
+        <span class="badge badge-light badge-pill text-success pb-0 mr-1"><?php echo sprintf(lang('note_active_jobs'), $totalActiveJobs); ?></span>
+        <span class="badge badge-light badge-pill text-info pb-0"><?php echo sprintf(lang('note_new_applicants'), $totalNewApplicants); ?></span>
+    </div>
+</div>
+</div>
+<!-- <hr class="mt-md-0"> -->
+<div class="row">
+    <div class="col-md-4 my-3">
+        <div class="card shadow">
+          <img class="card-img-top w-50 h-50 mx-auto mt-4" src="<?php echo url_assets_img('active_jobs.jpg'); ?>" alt="https://www.vectorstock.com/royalty-free-vector/resume-job-search-and-recruiting-concept-hr-vector-23382878">
```
Comment (princeryansy) [inline application/modules/hirer/views/home.php]: put static url content in a constant or a method.

File: application/modules/hirer/views/home.php [hunk 6]
```diff
@@ -1,2 +1,113 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-6">
+        <div class="card shadow shadow-lg">
+            <div class="jumbotron pb-0">
+                <?php echo lang('note_top_agencies'); ?>
+                <hr class="my-4">
+                <a class="btn btn-primary" href="<?php echo url_assets_img('top_agency.pdf'); ?>" target="_blank"role="button">Learn more</a>
```
Comment (princeryansy) [inline application/modules/hirer/views/home.php]: please take note that the link for the pdf file is still tentative.

**Comment by syriazapanta** [comment]
@princeryansy Please see updated changes. Also modified, home_lang file for your checking. Thank you!

File: application/language/english/hirer/home_lang.php [hunk 7]
```diff
@@ -2,33 +2,33 @@
...
+
+$lang['note_browse_resume'] = 'Browse Resume';
+$lang['comment_browse_resume'] = 'This shows the number of applicants applied on your job post';
+
+$lang['note_manpower'] = 'Employer Inquiries';
+$lang['note_manpower2'] = 'View Employer Inquiries';
+$lang['comment_manpower'] = 'This shows the inquiries you\'ve made for Workabroad.';
+
+
+$lang['note_top_agencies'] = 'Be <strong class="text-primary"><em>One</em></strong> of</p>
```
Comment (princeryansy) [inline application/language/english/hirer/home_lang.php]: please fix this line.

**Pull request merged**